src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain.lotto;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+  private static final int LOTTO_CNT = 6;
+  private final List<LottoNumber> numbers;
+
+  public LottoTicket(List<Integer> numbers) {
+    validate(numbers);
+    this.numbers = convert(numbers);
+  }
+
```

Comment:
```suggestion
  public LottoTicket(List<Integer> numbers) {
    this(convert(numbers));
  }

  public LottoTicket(List<LottoNumber> numbers) {
    validate(numbers);
    this.numbers = numbers;
  }
```

주 생성자도 추가한 후 위와 같이 구현해 본다.